roles/consul/templates/consul.json.j2

```diff
@@ -14,5 +14,9 @@
   "bootstrap_expect": {{ consul_bootstrap_expect }},
 {% endif %}
   "data_dir": "/var/lib/consul",
-  "ui_dir": "/opt/consul-ui"
+  "ui_dir": "/opt/consul-ui",
+  "recursors": [ {{ consul_retry_join }} ],
```

Comment:
When "consul_syslog" not enabled this is going to break due to the comma here ```"recursors": [ {{ consul_retry_join }} ],```yielding an invalid json.